docs/clients/promtail/configuration.md

```diff
@@ -1159,3 +1188,34 @@ scrape_configs:
       - source_labels: ['__syslog_message_hostname']
         target_label: 'host'
 ```
+
+## Example Push Config
+
+The example starts Promtail as a Push receiver and will accept logs from other Promtail instances or the Docker Logging Dirver:
+
+```yaml
+server:
+  http_listen_port: 9080
+  grpc_listen_port: 0
+
+positions:
+  filename: /tmp/positions.yaml
+
+clients:
+  - url: http://ip_or_hostname_where_Loki_run:3100/loki/api/v1/push
+
+scrape_configs:
+- job_name: push1
+  push:
```

Comment:
Just a thought, should we call this `listen` rather than `push`?

Reply:
I debated the naming of this, was also considering naming it `loki`.  The problem with `push` is it's not obvious it means it's listening, the problem with `listen` is it's a bit generic.

Hrm...

Reply:
ok, i renamed it to `loki_push_config`, wdyt @adityacs ?

Reply:
+1 for `loki_push_config`. We should see others opinion as well.